docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,6 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
-If it's not the case, please refer to the following section(#)
-
+If it's not the case, please refer to the following section(#)
```

Comment:
Suggestion: Good if you can explain this condition clearly. For example, under what condition or case, does the user need to refer to ta particular section.

Question: I do not see any sections - can you please help me identify what I am missing here?